meshtastic/interfaces/ble/compatibility_service.py

```
@@ -0,0 +1,380 @@
+                runnable = get_nowait()
+            except Empty:
+                break
+            iterations += 1
+            BLECompatibilityEventService._safe_execute(
+                iface,
+                runnable, error_msg="Error in deferred publish callback", reraise=False
+            )
+
+    @staticmethod
+    def publish_connection_status(
+        iface: "BLEInterface", connected: bool, *, publishing_thread: object
```

Ruff (0.15.5)

[warning] Boolean-typed positional argument in function definition

(FBT001)